acceptance/chaos_test.go

```diff
@@ -145,13 +159,33 @@ func TestChaos(t *testing.T) {
 			}
 			i++
 		case <-time.After(1 * time.Second):
-			// Periodically print out progress so that we know the test is still
-			// running.
-			cur := make([]string, num)
-			for i := range cur {
-				cur[i] = fmt.Sprintf("%d", atomic.LoadInt64(&counts[i]))
+			var newOutput string
+			if time.Now().Before(deadline) {
+				curRound := atomic.LoadInt64(&roundCopy)
+				if curRound == oldRound {
+					if time.Now().After(stallTime) {
+						close(stopper)
```

Comment:
you're technically not supposed to close `stopper` (it's racy, though a very unlikely race).